### ISSUE TITLE
Calendar Date buttons are not accessible using Keyboard navigation

### ISSUE THREAD

**Issue by Riya161999** [mark-tate/use-date-input#44]
While navigating if events are linked with calendar, then it is not accessible via "Enter" key. 
 We tried with "initialEnableKeyboardNavigation" but still not working.
 
Component used:  https://mark-tate.github.io/use-date-input/calendarProvider 
 
 Is there any way to make it accessible via "Enter" key??
